examples/demo-rollup/src/rng_xfers.rs

```diff
@@ -60,10 +63,10 @@ fn generate_create(start_nonce: u64) -> Vec<u8> {
         salt: 11,
         token_name: "sov-test-token".to_string(),
         initial_balance: 100000000,
-        minter_address: minter_address.clone(),
+        minter_address,
         authorized_minters: vec![minter_address],
     };
-    let enc_msg = Runtime::<DefaultContext>::encode_bank_call(msg);
+    let enc_msg = <Runtime<DefaultContext> as EncodeCall<Bank<DefaultContext>>>::encode_call(msg);
```

Comment:
Do I understand correctly that this change is related to https://github.com/Sovereign-Labs/sovereign-sdk/issues/389 ?

Reply:
Yes exactly